<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/icons.qrc">
    <normaloff>:/icons/64x64/configure.png</normaloff>:/icons/64x64/configure.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupbox_server_settings">
     <property name="title">
      <string>SQL Server</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="layout_server_settings">
        <item row="0" column="0">
         <widget class="QLabel" name="label_driver">
          <property name="text">
           <string>Driver</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="drivers"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_odbc_driver">
          <property name="text">
           <string>ODBC driver</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_server">
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="username"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_password">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="odbc_drivers"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="server"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_trusted_connection">
        <item>
         <widget class="QCheckBox" name="trusted_connection">
          <property name="text">
           <string>Trusted connection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_login_timeout">
        <item>
         <widget class="QLabel" name="label_login_timeout">
          <property name="text">
           <string>Login timeout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="login_timeout">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>900</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_login_timeout">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_paths">
     <property name="title">
      <string>Backup paths</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="layout_local_path">
        <item>
         <widget class="QLabel" name="label_local_directory">
          <property name="text">
           <string>Local path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="local_path"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_select_local_path">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_remote_path">
        <item>
         <widget class="QLabel" name="label_remote_path">
          <property name="text">
           <string>Remote path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remote_path"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_buttons">
     <item>
      <spacer name="spacer_buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_test">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Test server settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>drivers</tabstop>
  <tabstop>odbc_drivers</tabstop>
  <tabstop>server</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>trusted_connection</tabstop>
  <tabstop>login_timeout</tabstop>
  <tabstop>local_path</tabstop>
  <tabstop>remote_path</tabstop>
  <tabstop>button_select_local_path</tabstop>
  <tabstop>button_test</tabstop>
 </tabstops>
 <resources>
  <include location="../data/data.qrc"/>
  <include location="../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
